docker/build_jaxbtck.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash -x
 #
-# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2019,2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
I am not sure whether copyright plugin complains about the missing space in between years